features/command_line_interface/smells_count.feature

```diff
@@ -3,11 +3,12 @@ Feature: Reports total number of code smells
   Reek outputs the total number of smells among all files inspected.
 
   Scenario: Does not output total number of smells when inspecting single file
-    When I run reek spec/samples/standard_smelly/dirty.rb
+    Given a smelly file
+    When I run reek smelly.rb
```

Comment:
Same here - the problem is that you disconnect two things that belong together IMHO. "We have smelly file" shouldn't be separated from "what that is"...